src/imgix-core-js.js

```diff
@@ -158,7 +158,7 @@
     };
 
     ImgixClient.prototype._buildSrcSetPairs = function(path, params, options) {
-      var srcset = '';
```

Comment:
No change needed, but I would probably call this `srcsets` rather than just `srcset`